docs/intro/intro_symbolic_math.rst

```diff
@@ -0,0 +1,85 @@
+.. _intro_symbolic_math:
+
+=============
+Symbolic Math
+=============
+
+dimod enables easy incorporation of binary and integer variables as
+:ref:`single-variable models <symbolic_math_generators>`. For example, you can
+represent such binary variables as follows:
+
+>>> from dimod import Binary, Spin
+>>> x = Binary('x')
+>>> s = Spin('s')
+>>> x
+BinaryQuadraticModel({'x': 1.0}, {}, 0.0, 'BINARY')
+
+Similarly for integers:
+
+>>> from dimod import Integer
+>>> i = Integer('i')
+>>> i
+QuadraticModel({'i': 1.0}, {}, 0.0, {'i': 'INTEGER'}, dtype='float64')
+
+The construction of such variables as either BQMs or QMS depends on the type of
```

Comment:
Specifically, anything with more than one variable type or with integers will be a quadratic model.